target/linux/mpc85xx/image/Makefile

```diff
@@ -47,7 +47,8 @@ endef
 ifeq ($(SUBTARGET),generic)
 
 define Device/tl-wdr4900-v1
-  DEVICE_TITLE := TP-Link TL-WDR4900
+  DEVICE_VENDOR := TP-Link
+  DEVICE_MODEL := TL-WDR4900
```

Comment:
I suggest add ``DEVICE_VARIANT := v1``. Here You can find v2:
https://github.com/openwrt/openwrt/blob/dcb5cf85c8196b2ba35a0aa395f2247ed2163121/target/linux/ath79/image/generic-tp-link.mk#L249